docs/platforms/powershell/configuration/sampling.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Sampling
+description: "Learn how to configure the volume of error and transaction events sent to Sentry."
+sidebar_order: 50
+---
+
+Adding Sentry to your app gives you a great deal of very valuable information about errors and performance you wouldn't otherwise get. And lots of information is good -- as long as it's the right information, at a reasonable volume.
+
+## Sampling Error Events
+
+To send a representative sample of your errors to Sentry, set the <PlatformIdentifier name="sample-rate" /> option in your SDK configuration to a number between `0` (0% of errors sent) and `1` (100% of errors sent). This is a static rate, which will apply equally to all errors. For example, to sample 25% of your errors:
+
+<PlatformContent includePath="configuration/sample-rate" />
+
+The error sample rate defaults to `1`, meaning all errors are sent to Sentry.
+
+<Note>
+
+Changing the error sample rate requires re-deployment. In addition, setting an SDK sample rate limits visibility into the source of events. Setting a rate limit for your project (which only drops events when volume is high) may better suit your needs.
+
+</Note>
+
+## Sampling Transaction Events
+
+We recommend sampling your transactions for two reasons:
+
+1. Capturing a single trace involves minimal overhead, but capturing traces for _every_ page load or _every_ API request may add an undesirable load to your system.
+2. Enabling sampling allows you to better manage the number of events sent to Sentry, so you can tailor your volume to your organization's needs.
+
+Choose a sampling rate with the goal of finding a balance between performance and volume concerns with data accuracy. You don't want to collect _too_ much data, but you want to collect sufficient data from which to draw meaningful conclusions. If you’re not sure what rate to choose, start with a low value and gradually increase it as you learn more about your traffic patterns and volume.
+
+## Configuring the Transaction Sample Rate
+
+The Sentry SDKs have two configuration options to control the volume of transactions sent to Sentry, allowing you to take a representative sample:
+
+1. Uniform sample rate (<PlatformIdentifier name="traces-sample-rate" />):
+
+   - Provides an even cross-section of transactions, no matter where in your app or under what circumstances they occur.
+   - Uses default [inheritance](#inheritance) and [precedence](#precedence) behavior
+
+2. Sampling function (<PlatformIdentifier name="traces-sampler" />) which:
+   - Samples different transactions at different rates
+   - <PlatformLink to="/configuration/filtering/">Filters</PlatformLink> out some
+     transactions entirely
+   - Modifies default [precedence](#precedence) and [inheritance](#inheritance) behavior
```

Comment:
```suggestion
   - Modifies default [inheritance](#inheritance) and [precedence](#precedence) behavior
```

Reply:
Feel free to ignore these suggestions then!

Reply:
OK, closing these conversations